Documentation/DocuBlocks/Rest/AQL/JSF_get_api_aqlfunction.md

```diff
@@ -12,17 +12,51 @@ Returns all registered AQL user functions from namespace *namespace*.
 @RESTDESCRIPTION
 Returns all registered AQL user functions.
 
-The call will return a JSON array with all user functions found. Each user
-function will at least have the following attributes:
-
-- *name*: The fully qualified name of the user function
-
-- *code*: A string representation of the function body
+The call will return a JSON array with all user functions found.
```

Comment:
yes? 
```
{
	    code: 200,
		error: false,
		result: [
	      {
		    "name":"UnitTests::mytest1",
			"code":"function () { return 1; }",
			"isDeterministic":false
		}]}```

Reply:
I was asking because the docs claimed something contrary ("The call will return a JSON array with all user functions found" - this is true, but this is not the whole truth. The JSON array will be returned underneath the "result" attribute and not on the top-level).